src/database/index.tsx

```diff
@@ -257,11 +258,14 @@ export const saveScenario = async (scenario: any): Promise<Scenario | null> => {
 
 export const getFacilities = async (
   scenarioId: string,
-): Promise<Array<Facility> | null> => {
+): Promise<Facility[]> => {
```

Comment:
Just cleaning up this interface to be a little more consistent with our other methods that return arrays